src/adapters/poi_popup.js

```diff
@@ -72,7 +80,6 @@ PoiPopup.prototype.showPopup = function(poi, event) {
   const popupOptions = {
     className: 'poi_popup__container',
     closeButton: false,
-    closeOnClick: true,
```

Comment:
If we used `closeOnClick` or `closeOnMove`, we wouldn't have a chance to call `this.close()`, which would results in `this.popupHandle` not being cleared from memory.